changelog.d/13814.feature

```diff
@@ -0,0 +1 @@
+Keep track when we fail to process a pulled event over federation so we can intelligently back-off in the future.
```

Comment:
Same changelog as https://github.com/matrix-org/synapse/pull/13589 since we're still doing the same sort of thing. Just recording other exceptions here now.